holoviews/plotting/mpl/plot.py

```diff
@@ -280,7 +280,7 @@ def _update_title(self, key):
         title = self._format_title(key) if self.show_title else ''
         if 'title' in self.handles:
             self.handles['title'].set_text(title)
-        else:
+        elif self.handles['axis'].figure is not None:
```

Comment:
Let's be completely safe I guess:

```suggestion
        elif 'axis' in self.handles and self.handles['axis'].figure is not None:
```